src/effects/VST3/VST3ParametersWindow.cpp

```diff
@@ -255,6 +265,14 @@ VST3ParametersWindow::VST3ParametersWindow(wxWindow *parent,
       if((parameterInfo.flags & (Vst::ParameterInfo::kCanAutomate | Vst::ParameterInfo::kIsBypass | Vst::ParameterInfo::kIsReadOnly)) == 0)
          continue;
 
+      if(hideMIDICC)
+      {
+         static_assert(sizeof(Steinberg::tchar) == 2, "Make sure that UNICODE is defined in VST3");
+         static const std::u16string_view MIDI_CC { u"MIDI CC " };
+         if(std::u16string_view(parameterInfo.title).rfind(MIDI_CC, 0) == 0)
```

Comment:
Is this a reliable naming convention in all effects, or is this only a special case fix for one or a few problematic effects from one vendor?

Is there no other reliable way to decide this from the information in ParameterInfo?

Reply:
MIDI Control Change.

> Is this a reliable naming convention in all effects, or is this only a special case fix for one or a few problematic effects from one vendor?

JUCE framework uses such naming template. 

> Is there no other reliable way to decide this from the information in ParameterInfo?

I've spent some time reading the documentation and Steinberg forums and didn't find one.